src/CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 if(NOT CMAKE_INSTALL_RPATH)
   set(CMAKE_INSTALL_RPATH "${CEPH_INSTALL_FULL_PKGLIBDIR}")
 endif()
+# for rbdmap-generator
+set(CMAKE_INSTALL_SYSTEMD_GENERATORDIR "${CMAKE_INSTALL_LIBEXECDIR}/systemd/system-generators"
```

Comment:
again, please don't use CMAKE for our own variables.

Reply:
and `CMAKE_INSTALL_LIBEXECDIR` is proven to be wrong.

Reply:
Just want to be consistent with https://github.com/ceph/ceph/blob/b7e57a079ac0ba387b8c2ec7ad93df37c02fd90d/systemd/CMakeLists.txt#L1-L2

If you don't like that, I will fix the above code first. But I would like to first investigate it. So maybe I will update after some days.